cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc

```diff
@@ -43,13 +43,6 @@ While a composite index applies only to entities that match the indexed label an
 
 For information on how to configure full-text schema indexes, refer to <<operations-manual#index-configuration-fulltext-search,  Operations Manual -> Indexes to support full-text search>>.
 
-[NOTE]
```

Comment:
I would think that users of 4.0 would still need to know how to port from 3.x to 4.0. Is there another place in the documentation that covers this? I think it is good to remove these bits and pieces here and there if there is a section of porting from 3.x, but not good to remove this if no such content exists.

Reply:
That is a good point, and we should make sure there is a section about this in whatever "migrating to this new and shiny 4.0 release" document that will have to accompany the release. I have made a note of it in a card on our wall.